packages/ra-ui-materialui/src/button/SaveButton.tsx

```diff
@@ -57,16 +58,24 @@ export const SaveButton = <RecordType extends RaRecord = any>(
         onClick,
         mutationOptions,
         saving,
-        disabled,
+        disabled: disabledProp,
         type = 'submit',
         transform,
         variant = 'contained',
+        alwaysEnable,
```

Comment:
since you're testing -== false down there, you should give it a default value of false here